config/locales/en.yml

```diff
@@ -234,6 +236,16 @@ en:
               blank: You must provide a provider name
```

Comment:
Not your code! But I prefer your wording :)

For consistency could you change this to `"You must enter a provider name"`

Thanks!